packages/suite/src/components/suite/FormFractionButtons.tsx

```diff
@@ -51,6 +51,8 @@ export const FormFractionButtons = ({
     const isMaxDisabled =
         isDisabled || new BigNumber(totalAmount || '0').lt(MIN_ETH_AMOUNT_FOR_STAKING);
 
+    const displaySymbol = getNetworkDisplaySymbol(symbol).toUpperCase();
```

Comment:
Why call `toUpperCase` everywhere and not just have it inside `getNetworkDisplaySymbol`?

Reply:
Yes, it would be better.

Reply:
However, there are some places where lowercase is necessary. We could stay with lowercase to maintain the consistency of the symbols - same as `symbol`. What do you think?

Reply:
I agree with @martykan.
However, what I would do is that I would specify it in the correct form in networksConfig.

`ETH`, `BNB`,... the symbol can be mixed cased (I dont know such a network but I know such tokens) so we should not uppercase/lowercase it if not necessary

Reply:
I think that on device we show `tHOL` but not 100% sure